CMakeLists.txt

```diff
@@ -62,6 +62,8 @@ project(srf
   LANGUAGES C CXX
 )
 
+rapids_cmake_write_version_file(${CMAKE_CURRENT_SOURCE_DIR}/include/srf/version.hpp)
```

Comment:
I think this + the .gitignore entry results in the same functional result; but I agree it might make more sense to have all generated artifacts live in the build directory.

Reply:
Lets leave it where it is for now. If we need to move it in the future, we can. This does keep things simple and is effectively the same

Reply:
Updated this to create the version file in the build directory.